var/spack/repos/builtin/packages/ganglia/package.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Ganglia(AutotoolsPackage):
+    """Ganglia is a scalable distributed monitoring system for high-performance
+    computing systems such as clusters and Grids. It is based on a hierarchical
+    design targeted at federations of clusters. Supports clusters up to 2000
+    nodes in size."""
+
+    homepage = "http://ganglia.sourceforge.net/"
+    url      = "http://jaist.dl.sourceforge.net/project/ganglia/ganglia%20monitoring%20core/3.7.2/ganglia-3.7.2.tar.gz"
+    list_url = "http://jaist.dl.sourceforge.net/project/ganglia/ganglia%20monitoring%20core"
+    list_depth = 1
+
+    version('3.7.2', sha256='042dbcaf580a661b55ae4d9f9b3566230b2232169a0898e91a797a4c61888409')
+
+    depends_on('m4',       type='build')
+    depends_on('autoconf', type='build')
+    depends_on('automake', type='build')
+    depends_on('libtool',  type='build')
+    depends_on('apr')
+    depends_on('libconfuse')
+    depends_on('python@:2.7.999')
```

Comment:
Probably type=build/run?

Reply:
Emm, it need link to python:
```
  >> 903    /usr/bin/ld: cannot find -lpython2.7
  >> 904    collect2: error: ld returned 1 exit status
     905    make[4]: *** [Makefile:378: modpython.la] Error 1
```
but I also find another issue, it depends on libexpact. 
So I fixed it in https://github.com/spack/spack/pull/16382/commits/40b400002bbd950999772ac03208f11fc02ed1cb